lib/transports/sshTransport.js

```diff
@@ -65,6 +56,7 @@ function sshCall(config, xmlIn, done) {
         if (verbose) {
           console.log('Exec error: ', error);
         }
+        client.end();
```

Comment:
Looks like `client.end()` will check that it's [connected](https://github.com/mscdex/ssh2/blob/master/lib/client.js#L704) first, so we can move these calls in the error cases to the error handler itself and it will do the right thing regardless of if it is connected or not.

Reply:
Ok will move `client.end()` calls to the error handler.